lib/connection.js

```diff
@@ -82,8 +82,13 @@ Connection.prototype.connect = function (port, host) {
 
   this.stream.once('data', function (buffer) {
     var responseCode = buffer.toString('utf8')
-    if (responseCode !== 'S') {
-      return self.emit('error', new Error('The server does not support SSL connections'))
+    switch (responseCode) {
+      case 'N':
+        return self.emit('error', new Error('The server does not support SSL connections'))
+      case 'S':
+        break
+      default:
```

Comment:
perhaps adding a short comment for every case would be useful, these codes are not well known for many users.

Reply:
Agreed, done.

Reply:
Thx !